go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/scylladb/scylla-manager/v3
 go 1.23.2
 
 require (
+	cloud.google.com/go/compute/metadata v0.3.0
```

Comment:
Going with the `squash and merge` and mixing commits that update vendor and implement features is messy. In order to make it cleaner, we can either:
- make a separate PR for updating vendor
- don't use `squash and merge` - the owner of the PR would need to manually squash commits with some reasonable logic (e.g. separate vendor changes from feature implementations) and then use the `rebase and merge` option

Reply:
Yes, I can squash changes manually before merging and then merge without squashing.